components/UsaGovHeader.vue

```diff
@@ -459,20 +459,33 @@
           class="back-benefit font-serif-md">
           Benefits, Grants, Loans
         </a>
-        <a
+        <button
           class="language-toggle-mobile"
-          href="https://www.usa.gov/espanol/"
           lang="es"
-          xml:lang="es">
-          Español
-        </a>
+          xml:lang="es"
+          @click="changeLanguage()">
+          {{ this.$i18n.locale === "en" ? "Español" : "English" }}
```

Comment:
think I could use this for routing purposes

Reply:
Yes, depends on where it is being used, seems like in HTML it is accessed via `this.$i18n` in other places such as asyncData functions that have another parameter inside, it would be added as a parameter like such: `i18n`. In all other instances it is referenced at `this.i18n`